src/app/create/create.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="create">
+  <h1>Create An Account</h1>
+  <br>
+  <form novalidate class="simple-form">
+    <label>
+      <input placeholder="Username" type="text" ng-model="user.username" />
```

Comment:
@rgavinc The backend is only expecting an email and password, so that we don't have to a unique username as well as a unique email. 😄

Reply:
Ok, I took out the username input